my-bq-app/src/firebase/credentials.js

```diff
@@ -0,0 +1,24 @@
+// import function to initialize firebase app
```

Comment:
Remember to add this to your git ignore file (:

Reply:
thank you Perla, yes I forgot.